protocol/protocolwrapper/protocol_filter_wrapper.go

```diff
@@ -68,21 +68,16 @@ func (pfw *ProtocolFilterWrapper) Destroy() {
 }
 
 func buildInvokerChain(invoker protocol.Invoker, key string) protocol.Invoker {
-	filtName := invoker.GetUrl().GetParam(key, "")
-	if filtName == "" {
```

Comment:
why do you remove this `if`?

Reply:
If filterName equals "", then filterNames will equal to []. Then because filterNames is empty string slice, and len(filterNames) equals zero, the circle will return immediately.

Reply:
In this case, `filterNames` is related to `filterName` .

If some day, they dont have a relationship. I think it will cause a a hidden problem in the future...

What do you think? @fangyincheng @AlexStocks

Reply:
I revert this code.